src/main/java/org/schemaspy/output/xml/dom/XmlProducerUsingDOM.java

```diff
@@ -96,6 +97,7 @@ public void generate(Database database, File outputDir) {
 
         xmlTableFormatter.appendTables(rootNode, tables);
         xmlRoutineFormatter.appendRoutines(rootNode, database.getRoutines());
+        xmlSequenceFormatter.appendSequences(rootNode, database.getSequences());
```

Comment:
I think i want this before tables or possibly before routines.
Since I believe that tables and/or routines might use Sequences.

Reply:
ok